src/pages/User/Home.tsx

```diff
@@ -1,14 +1,65 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
-import AuthenticationButton from '@components/user/AuthenticationButton';
+import Container from '@components/common/container/Container';
+import styled from '@emotion/styled';
+import ArrowRight from '../../resources/svg/ArrowRight';
+import NavBar from '@components/common/nav/NavBar';
+
+const MainTitle = styled.div`
+  width: 540px;
+  height: 160px;
+  font-size: 64px;
+  font-weight: 800;
+  margin-bottom: 24px;
+`;
+
+const LinkAdminHome = styled(Link)`
+  text-decoration: none;
+  width: 348.695px;
+  height: 80px;
+  border-radius: 50px;
+  background: #eb6d09;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+
+  &:hover {
+    background: linear-gradient(133deg, #ff9f32, #eb6d09);
+  }
+`;
+
+const LinkText = styled.div`
+  width: 314.256px;
```

Comment:
```suggestion
  width: 315px;
```

디자인에 소수점으로 되어있어도 정수로 적당히 바꿔주세요